### ISSUE TITLE
Extra header added when import directly follows header

### ISSUE THREAD

**Issue by dcutting** [nicklockwood/SwiftFormat#441]
```
//  Copyright © 2019 Yellowbek. All rights reserved.
import Foundation
```

formats to:

```
//  Copyright © 2019 Yellowbek. All rights reserved.

//  Copyright © 2019 Yellowbek. All rights reserved.
import Foundation
```

**Comment by nicklockwood** [comment]
This is kind if a known issue (it's not possible in general to distinguish between header comments and regular comments unless they are followed by a blank line), but in this particular instance it should be fixable since I can just check if the first comment matches the header template.

I'll try to get this fixed for the next release.

**Comment by nicklockwood** [comment]
@dcutting fixed in 0.40.10.
